tests/Demo.py

```diff
@@ -263,17 +277,19 @@ def Main(args=None):
 
     try:
         config = pconfig.parse_configuration_files(conffiles, confpaths)
-        json.dumps(config, indent=4)
+        config_json_str = json.dumps(config, indent=4)
```

Comment:
Looks like config_json_str is not used. Will pylint report unused variable issue ?

Reply:
You're right. Looks like it was there for debugging. I commented it out.

Reply:
I think the line " json.dumps(config, indent=4)" was added to check if config is a well-formatted json string else it will throw an exception. So this line was not added just for debugging. So this line shouldn't be commented out.

Reply:
FIxed.  The `json.dumps` line is now unchanged.